pull.go

```diff
@@ -24,23 +24,28 @@ func (cmd *PullCommand) Run() error {
 		Debug = true
 	}
 
-	client, err := phraseapp.NewClient(cmd.Config.Credentials)
-	if err != nil {
-		return err
-	}
-
-	targets, err := TargetsFromConfig(cmd)
-	if err != nil {
-		return err
-	}
+	err := func() error {
```

Comment:
what is the reason for this anonymous function? looks like it would behave 1:1 the same without it?